index.html

```diff
@@ -10,43 +10,52 @@
 </head>
 
 <body>
-    <main class="main-section">
 
         <header>
             <nav>
+                <ul class="top-nav">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
                 <img src="images/burgers.svg" alt="Menu-bar" class="hamburgers">
```

Comment:
- [ ] I would suggest you to stick to the design as must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)... kindly fix the background image to be position at the top of the page you can give it a little margin top to look like the original figma design.
- [ ] your design

![Screenshot 2023-06-15 184924](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/4226123d-6062-406f-913f-a7ee1fc065ab)

- [ ] figma desig 
![fig design3](https://github.com/Young-Gaius/portfolio-setup/assets/92458236/7b0c5eff-e821-45ee-9aa6-72c5823f3a48)